pkg/KubeArmorOperator/internal/controller/cluster.go

```diff
@@ -1222,7 +1223,7 @@ func (clusterWatcher *ClusterWatcher) WatchRecommendedPolicies() error {
 				clusterWatcher.Log.Info("created csp", csp.GetName())
 			}
 		case false:
-			if !policy.IsDir() {
+			if !policy.IsDir() && csp.Annotations["app.kubernetes.io/managed-by"] == "kubearmor-operator" {
```

Comment:
can we keep it like 
```suggestion
			if !policy.IsDir() && csp.Annotations["kubearmor.kubernetes.io/managed-by"] == "kubearmor-operator" {
```
it would be in line with https://github.com/kubearmor/KubeArmor/blob/main/pkg/KubeArmorController/common/common.go#L18
wdyt @rksharma95 @rootxrishabh